hardware_drivers/roboteq/roboteq_driver/mbs/script.hex

```diff
@@ -1,38 +1,38 @@
 :10000000021C01060201020106027F0F0202010921
 :10001000060206020102010F29020901020127025C
 :10002000080103E803270213020102012702130259
-:1000300002020127020201032C012702030102C868
-:100040002702010204060204050201051D03A8009F
-:100050001F024802040501270249020405030004A7
-:10006000270236020405026427023802040501272C
-:10007000022A02040503580227022D02040503E8A0
-:100080000327022B0204050326022702330204057C
-:10009000039001270234020405039001270204059E
-:1000A00014020406034600200201020406020405AD
-:1000B0000201051D0389011F020202040502010A53
-:1000C000090502060204052A19037A011F02060225
-:1000D00004052A020202040502010A0906020602B8
-:1000E00004052A02010F180314011F022702040548
-:1000F0000127022E0204050127022F020405012711
-:100100000230020405012702320204050127010220
-:100110000405012902060204052A01180347011FEC
-:1001200002270204050127022E02040502142702F9
-:100130002F0204050228270230020405020A2702C2
-:100140003202040502642702060204052A0201188D
-:10015000037A011F02270204050127022E0204056B
-:10016000020227022F0204050202270230020405C0
-:100170000127023202040502642703A901210204B7
-:10018000051402040603AD0020020505140205064D
-:1001900002050502051803A2011F03110221010235
-:1001A000050602132303A80020240326663A02044E
-:1001B000052524013A010204052A2524013A0201F9
-:1001C0000204052A2524013A02020204052A2524F4
-:1001D000013A02030204052A2524013A020402041A
-:1001E000052A2524013A020D02022A2524013A0299
-:1001F0000C0204052A2524013A02100204052A25CE
-:1002000024013A02120204050201092A2524010DE3
-:1002100022240326733A01052524013A0215012AF6
-:100220002524013A0214012A2524013A020D020173
-:100230002A2524013A020D02032A2524013A02123A
-:1002400002012A2524010D2200FFFFFFFFFFFFFF0F
+:1000300002020127020201032C012702030102B47C

```

Comment:
It's the .mbs script compiled into hex, as per their instructions on [this readme](https://github.com/uf-mil/Navigator/blob/master/hardware_drivers/roboteq/roboteq_driver/mbs/README.md). It's a configuration flashed to each thruster on startup